src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -174,6 +180,15 @@ export type RectangleBounds = {
     yStart?: number;
 };
 
+export type MolecularProfileOption = {
+    value: string;
+    count: number;
+    label: string;
+    description: string;
+    sampleUniqueKeys: string[];
+    dataType?: string;
```

Comment:
dataType is optional, this is to support `Generic Assay`. `Generic Assay` has three dataType: `CONTINUOUS`, `CATEGORY` and `BINARY`, different dataType will have different chart type. For example, `CONTINUOUS` should create a bar chart while `CATEGORY` and `BINARY` should create a pie chart.

Reply:
maybe we should set this dataType to enum to enforce this?

Reply:
@alisman backend didn't expose the `dataType`, maybe we should leave as it? Or we may need to create one and update along with backend?